### PR TITLE
CV Template: Implement collapsible metadata

### PR DESCRIPTION
**Pull request by DunklesArchipel**
I do not know how to convert the package into a `dict`. So currently it is `entry.package.to_dict()`, however, `to_dict` seems to be deprecated. There should be another solution to the problem.

### CODE REVIEW

**Comment by DunklesArchipel** [comment]
> I do not know how to convert the package into a `dict`. So currently it is `entry.package.to_dict()`, however, `to_dict` seems to be deprecated. There should be another solution to the problem.

I found the workaround.

File: templates/cv_entry.md [hunk 1]
```diff
@@ -27,18 +27,17 @@ Properties of the electrochemical cell:
...
-## electrolyte
-0.1 M CsF + 0.1 M HClO4
-
-## reference electrode
-
-{{ entry.electrochemical_system.electrodes.reference_electrode.type }}
-
-## scan rate
-{{ entry.figure_description.scan_rate.value }} {{ entry.figure_description.scan_rate.unit | lower }}
+<!-- TODO: These sections should be removed. See #31. -->
```
Comment (saraedum) [inline templates/cv_entry.md:43]:
```suggestion
```

File: echemdb/data/cv/entry.py [hunk 2]
```diff
@@ -296,3 +296,16 @@ def __repr__(self):
 
         """
         return repr(self._descriptor)
+
+    def create_yaml(self):
```
Comment (saraedum) [inline echemdb/data/cv/entry.py:300]:
```suggestion
    @property
    def yaml(self):
```

**Review by saraedum — APPROVED** [review]
Looks good to me. But there are some merge conflicts.

**Pull request merged**